packages/segmented-control/src/SegmentedControlOption.tsx

```diff
@@ -327,38 +332,34 @@ const SegmentedControlOption = React.forwardRef<
         ref={forwardedRef}
         data-lg-checked={checked}
       >
-        <InteractionRing
-          darkMode={mode === 'dark'}
-          borderRadius={'var(--seg-ctrl-border-radius)'}
-          className={interactionRingStyle}
-        >
-          <Box as={as} tabIndex={-1} className={boxStyle} {...rest}>
-            <button
-              role="tab"
-              id={id}
-              tabIndex={focused ? 0 : -1}
-              aria-selected={checked}
-              aria-controls={ariaControls}
-              disabled={disabled}
-              className={buttonStyle}
-              ref={buttonRef}
-              onClick={onClick}
-              onMouseEnter={onMouseEnter}
-              onMouseLeave={onMouseLeave}
+        <Box as={as} tabIndex={-1} className={boxStyle} {...rest}>
```

Comment:
I was thinking of removing interaction ring for my components but didn't want to screw up the darkMode style but good to see that it worked in this case.